recipe/build.sh

```diff
@@ -3,19 +3,28 @@
 mkdir build
 cd build
 
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [[ ${HOST} =~ .*darwin.* ]]; then
+    CMAKE_PLATFORM_FLAGS+=(-DCMAKE_OSX_SYSROOT="${CONDA_BUILD_SYSROOT}")
```

Comment:
we can probably also just export `SDKROOT`: https://github.com/conda-forge/toolchain-feedstock/pull/47

Reply:
toolchain 2.3.0 adds this automatically now

Reply:
Yep, recently published. HTH :)

Reply:
How can I pull those in, just re-render?

Reply:
No, exporting `SDKROOT` does nothing @kalefranz. We use `CONDA_BUILD_SYSROOT` as do CF (once the new transition is complete that toolchain-feedstock is unused). I don't like needless env. pollution and duplicating this is that. The question is why would we do that?

Our compilers have `CONDA_BUILD_SYSROOT` *baked* into them. Use it.

Reply:
We do not need CMake to do anything for us wrt `SDKROOT`. We need no `-isysroot` flags passed for example.

Reply:
> exporting SDKROOT does nothing

Just to clarify, with regards to CMake an export of `SDKROOT` does set the default value for `CMAKE_OSX_SYSROOT`: https://cmake.org/cmake/help/v3.0/variable/CMAKE_OSX_SYSROOT.html

Reply:
> How can I pull those in, just re-render?

@ax3l, should just happen automatically.

> No, exporting SDKROOT does nothing...

@mingwandroid, taken from the CMake docs on [`CMAKE_OSX_SYSROOT`]( https://cmake.org/cmake/help/v3.13/variable/CMAKE_OSX_SYSROOT.html ):

> If not set explicitly the value is initialized by the SDKROOT environment variable, if set, and otherwise computed based on the CMAKE_OSX_DEPLOYMENT_TARGET or the host platform.

Anyways this is in `toolchain` so individual recipes shouldn't need it is the hope.